nuxeo-features/nuxeo-platform-rendition/nuxeo-platform-rendition-core/src/test/resources/test-default-rendition-override-contrib.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.ecm.platform.rendition.test.defaultOverride">
+
+  <require>org.nuxeo.ecm.platform.rendition.download.contrib</require>
+  <require>org.nuxeo.ecm.platform.rendition.publish.contrib</require>
+
+  <extension target="org.nuxeo.ecm.platform.rendition.service.RenditionService"
+    point="defaultRendition">
+    <defaultRendition reason="publish">
+      <script language="JavaScript">
+        function run() {
+          if (Document.getType() == "Folder") {
+            return 'zipTreeExport';
+          }
+          return null;
```

Comment:
nit: can we omit the return here just to be sure we're handling implicit default return value of `undefined`?